codalab/lib/bundle_store.py

```diff
@@ -74,11 +54,320 @@ def get_bundle_location(self, data_hash):
         """
         pass
 
-    def reset(self):
+class BalancedMultiDiskBundleStore(BaseBundleStore, BundleStoreCleanupMixin, BundleStoreHealthCheckMixin):
```

Comment:
Alright, I was planning to do that once we switched over, but I'll do it in this PR directly.